### PR TITLE
fix: workaround the issue with e2e tests

### PR DESCRIPTION
**Pull request by hsiliev**
Swap scenario 4 and 5
Do not use uuid as steps already have unique IDs

### CODE REVIEW

File: .github/workflows/run-e2e.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ jobs:
           publish_branch: gh-pages/dimpact-e2e-test-report # deploying to gh-pages branch
           keep_files: true # retains files that are not part of the current publish
       - name: Notify Slack
-        if: always()
+        if: ${{ github.ref == 'refs/heads/master' }}
```
Comment (edgarvonk) [inline .github/workflows/run-e2e.yml:60]: We don't use `master`. Nobody should. ;-) It is `main` we use.

**Review by edgarvonk — CHANGES_REQUESTED** [review]

File: .github/workflows/run-e2e.yml (hunk 1 above)
Comment (edgarvonk) [reply .github/workflows/run-e2e.yml:60]: Also the `${{ }}` are not needed I think.

Comment (hsiliev) [reply .github/workflows/run-e2e.yml:60]: Copy-past**a** error

Comment (hsiliev) [reply .github/workflows/run-e2e.yml]: Fixed

File: src/e2e/step-definitions/authentication.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { When } from "@cucumber/cucumber";
 import { CustomWorld, authFile } from "../support/worlds/world";
 import { worldUsers } from "../utils/schemes";
 
-const LOGIN_TIMEOUT_MS = 20000;
+const ONE_MINUTE_IN_MS = 60 * 1000;
```
Comment (bas-info-nl) [inline src/e2e/step-definitions/authentication.ts:10]:
```suggestion
const ONE_MINUTE_IN_MS = 60_1000;
```

File: src/e2e/step-definitions/hooks.ts [hunk 3]
```diff
@@ -1,14 +1,15 @@
 import { After, AfterAll, AfterStep, Before, Status } from "@cucumber/cucumber";
 import { CustomWorld, authFile } from "../support/worlds/world"
-import { v4 as uuidv4 } from 'uuid';
 import fs from 'fs'
 import { testStorageFile } from "../utils/TestStorage.service";
 
+const ONE_MINUTE_IN_MS = 60 * 1000;
```
Comment (bas-info-nl) [inline src/e2e/step-definitions/hooks.ts:6]:
```suggestion
const ONE_MINUTE_IN_MS = 60_1000;
```

**Review by bas-info-nl — APPROVED** [review]

File: src/e2e/step-definitions/hooks.ts (hunk 3 above)
Comment (hsiliev) [reply src/e2e/step-definitions/hooks.ts]: Fixed to `60_000`

**Pull request merged**